x-pack/plugins/uptime/public/state/selectors/index.ts

```diff
@@ -94,3 +94,8 @@ export const searchTextSelector = ({ ui: { searchText } }: AppState) => searchTe
 export const selectedFiltersSelector = ({ selectedFilters }: AppState) => selectedFilters;
 
 export const monitorIdSelector = ({ ui: { monitorId } }: AppState) => monitorId;
+
+export const journeySelector = ({ journeys }: AppState) => {
+  console.log('journeys', journeys);
```

Comment:
We need to remove this console.log statement

Reply:
I kinda like it /s

This is leftover from the demo I gave you this morning. I did a quick `add . && commit` for a checkpoint and forgot to reset it and review what I was merging.

Reply:
see https://github.com/elastic/kibana/pull/77960/commits/a6764f267f21bd8dd4fca3b6b70032c247825f95